frontend/packages/schema-editor/src/components/SchemaInspector/SchemaInspector.tsx

```diff
@@ -20,24 +19,19 @@ export const SchemaInspector = () => {
   }
 
   const selectedItem: UiSchemaNode = savableModel.getNodeByUniquePointer(selectedUniquePointer);
-  const shouldDisplayFieldsTab = isField(selectedItem) && isObject(selectedItem);
 
   return (
-    <Tabs defaultValue={t('schema_editor.properties')} className={classes.root}>
+    <Tabs defaultValue='Properties' className={classes.root}>
       <Tabs.List>
-        <Tabs.Tab value={t('schema_editor.properties')}>{t('schema_editor.properties')}</Tabs.Tab>
-        <Tabs.Tab value={t('schema_editor.fields')}>{t('schema_editor.fields')}</Tabs.Tab>
+        <Tabs.Tab value='Properties'>{t('schema_editor.properties')}</Tabs.Tab>
+        <Tabs.Tab value='Fields'>{t('schema_editor.fields')}</Tabs.Tab>
```

Comment:
Hvorfor endre dette til hardkodet tekst?

Reply:
Ikke noe stor sak egt, men tenkte det var misvisende at det var oversatte tekster her ettersom det kun er verdier som brukes i koden og ikke visuelt 🤷 Kan evt lage en enum for det slik vi gjør andre steder vi bruker tabs? 😄

Reply:
Supert!